### PR TITLE
VostFree Module

### PR DESCRIPTION
**Pull request by Bilna95**
create VostFree module folder

### CODE REVIEW

**Comment by mprotmod** [comment]
add the module to the index.html follow the instructions of the repo.

**Comment by Bilna95** [comment]
I made an another pull request for the index.html.

**Comment by mprotmod** [comment]
i dont see the commit, make sure you pushed it.

**Comment by Bilna95** [comment]
It should be ok now.

**Comment by mprotmod** [comment]
you have to set the update site to the .json link of this repo on the .json module and on the html index

File: index.html [hunk 1]
```diff
@@ -292,6 +292,24 @@ <h1 style="color: white; padding-left: 1rem; margin: auto;" class="zero">KETSU M
...
+    <img class="shadow maxWidth rounded absolute img-fluid"
+        src="https://i.imgur.com/AYyMYiE.png"
+        alt="VostFree Module">
+    <div style="margin-left: 6.5rem; margin-right: 6.5rem;" class=" fullHeight">
+        <p><strong>VostFree | FR </strong></p>
+        <p class="desc">Version: 1.2 | Developer: ナビル</p>
+    </div>
+    <div class="fullHeight addButton container-fluid fullHeight">
+        <div class="row align-items-center fullHeight">
+            <a href="ketsuapp://?moduleData=https://raw.githubusercontent.com/Bilna95/Modules-KETSU/main/Module%20Parser%20KETSU/VostFree/moduleTemplate.json">
```
Comment (mprotmod) [inline index.html:364]: Change that link to the link of this repository.

**Review by mprotmod — COMMENTED** [review]
add the link on there and on the UpdateSite on the moduleTemplate.json  make sure the link is of this repository not yours.

**Comment by Bilna95** [comment]
I can't put a link that I don't know

**Comment by Bilna95** [comment]
It should be fine now

**Comment by mprotmod** [comment]
Be more carefull because you messed up the html, i had to fix a bunch of stuff, dont modify the template code provided on the readme.

**Pull request merged**